bot.py

```diff
@@ -0,0 +1,214 @@
+import os
+import re
+import sys
+from datetime import datetime
+import time
+
+import discord
+import dotenv
+import redis
+from discord.ext import commands
+from discord_slash import SlashCommand
+from discord_slash.utils.manage_commands import create_option, create_choice
+from dotenv import dotenv_values
+
+r = redis.Redis(host=dotenv_values("config.env")["HOSTNAME"], port=dotenv_values("config.env")["PORT"], db=dotenv_values("config.env")["DB"])
+
+now = datetime.now()  # current date and time
+date_time = now.strftime("%m/%d/%Y, %H:%M:%S")
+
+whitelisted = dotenv_values("config.env")["whitelisted"].split(",")
+bot = commands.Bot(command_prefix="redis ")
+slash = SlashCommand(bot, override_type=True, sync_commands=True)
+
+guild_ids = [int(dotenv_values("config.env")["guild_id"])]
+
+
+async def env_adj(key, value):
+    dotenv.set_key("config.env", str(key).upper(), value)
+
+
+@slash.slash(name="Reboot", description=f"", guild_ids=guild_ids)
+async def _reboot(ctx):
+    now = datetime.now()  # current date and time
+    result = discord.Embed(
+        title='Rebooting',
+        description=f'Execution time: {now}\n\n**ATTEMPTING REBOOT**',
+        colour=discord.Colour.teal()
+    )
+    result.set_footer(text="This is an open source project by @YonLiud at GitHub", icon_url="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png")
+    await ctx.send(embed=result)
+    time.sleep(2)
+    try:
+        if str(ctx.author.id) not in whitelisted:
+            raise Exception("01 Permission Denied! User id is not found in whitelisted")
+        output = "Verification Complete!\nRebooting..."
```

Comment:
why would you do that in that way? why not simple else if? why raise exceptions? exceptions are for critical *state* errors, not auth